src/test/java/com/tictactoe/BoardTest.java

```diff
@@ -1,10 +1,10 @@
 package com.tictactoe;
 
-import com.tictactoe.Board;
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
```

Comment:
Nice. I like this change of testing against the array rather than the output string.